packages/eui/src/components/form/form_control_layout/form_control_layout_delimited.stories.tsx

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import {
+  hideStorybookControls,
+  moveStorybookControlsToCategory,
+} from '../../../../.storybook/utils';
+import { EuiIcon } from '../../icon';
+
+import {
+  EuiFormControlLayoutDelimited,
+  EuiFormControlLayoutDelimitedProps,
+} from './form_control_layout_delimited';
+import { EuiFormControlLayoutProps } from './form_control_layout';
+
+// re-declaring the component with props as the Partial<EuiFormControlLayoutProps>
+// of EuiFormControlLayoutDelimitedProps is otherwise not resolved
+const Component: FunctionComponent<
+  EuiFormControlLayoutDelimitedProps & EuiFormControlLayoutProps
+> = EuiFormControlLayoutDelimited;
+
+const meta: Meta<EuiFormControlLayoutDelimitedProps> = {
+  title: 'Forms/EuiForm/EuiFormControlLayoutDelimited',
+  component: Component,
+  argTypes: {
+    delimiter: {
+      control: 'radio',
+      options: [undefined, 'icon', 'text'],
+      mapping: {
+        icon: <EuiIcon type="arrowRight" />,
+        text: 'multiply',
```

Comment:
```suggestion
        text: '+',
```

Would prefer a shorter text if possible, I think that's going to be closer to production usage

Reply:
I'm fine with that. My thought here was that + might look like an icon and I wanted to ensure it's clear it's text 😅

Reply:
There's special styling I need to apply for text delimiters vs icons so it'll (sadly) be very clear that it's text if it breaks 😭